src/Illuminate/Console/Scheduling/Event.php

```diff
@@ -146,6 +153,7 @@ public function __construct(Mutex $mutex, $command)
         $this->mutex = $mutex;
         $this->command = $command;
         $this->output = $this->getDefaultOutput();
+        $this->timestamp = Carbon::now();
```

Comment:
Why set this is as a variable when it is always Carbon::now() and you could have just left its instantiation in the method itself like we had it?